WARNING  conftest:conftest.py:36 M       pages/multibranch_pipeline_config_page.py
WARNING  conftest:conftest.py:36 M       tests/multibranch_pipeline_configuration/test_change_properties.py
INFO     core.jenkins_utils:jenkins_utils.py:210 running cleanup
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 403 None
DEBUG    core.jenkins_utils:jenkins_utils.py:92 failed to get page on the first try, retrying after login
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /login?from=%2F HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /j_spring_security_check HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/securityRealm/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /computer/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/credentials/store/system/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /me/my-views/view/all/submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/appearance/configSubmit HTTP/1.1" 200 134
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /user/admin/security/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /crumbIssuer/api/json HTTP/1.1" 200 155
INFO     tests.api.support.base_api:base_api.py:52 Generating Jenkins API token at: "/generateNewToken"
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/descriptorByName/jenkins.security.ApiTokenProperty/generateNewToken HTTP/1.1" 200 169
INFO     tests.api.support.base_api:base_api.py:63 Generated token: 113a...
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /createItem?name=My_Multi_Project HTTP/1.1" 200 0
INFO     tests.api.support.jenkins_client:jenkins_client.py:23 [POST /createItem?name=My_Multi_Project] Status: 200 - OK
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /createItem?name=Test_Folder HTTP/1.1" 200 0
INFO     tests.api.support.jenkins_client:jenkins_client.py:23 [POST /createItem?name=Test_Folder] Status: 200 - OK
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /job/Test_Folder/createItem?name=Pipeline_one HTTP/1.1" 200 0
INFO     tests.api.support.jenkins_client:jenkins_client.py:23 [POST /job/Test_Folder/createItem?name=Pipeline_one] Status: 200 - OK
DEBUG    conftest:conftest.py:90 Argument --remote-allow-origins=* added to the chrome
DEBUG    conftest:conftest.py:90 Argument --disable-gpu added to the chrome
DEBUG    conftest:conftest.py:90 Argument --no-sandbox added to the chrome
DEBUG    conftest:conftest.py:90 Argument --disable-dev-shm-usage added to the chrome
DEBUG    conftest:conftest.py:90 Argument --headless=new added to the chrome
DEBUG    conftest:conftest.py:90 Argument --window-size=1920,1080 added to the chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:101 Selenium Manager binary found at: /opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:114 Executing process: /opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager --browser chrome --debug --language-binding python --output json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Sending stats to Plausible: Props { browser: "chrome", browser_version: "", os: "linux", arch: "x86_64", lang: "python", selenium_version: "4.31" }
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Found chromedriver 137.0.7151.119 in PATH: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chrome detected at /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Running command: /usr/bin/google-chrome --version
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Output: "Google Chrome 137.0.7151.119 "
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Detected browser: chrome 137.0.7151.119
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Discovering versions from https://googlechromelabs.github.io/chrome-for-testing/known-good-versions-with-downloads.json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Required driver: chromedriver 137.0.7151.119
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Driver path: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Browser path: /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.service:service.py:231 Started executable: `/usr/bin/chromedriver` in a child process with pid: 2760 using 0 to output -3
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:43355
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:43355 "POST /session HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:43355 "POST /session/1fd13aac053ff3a23e7ebaea4053fd68/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:43355 "GET /session/1fd13aac053ff3a23e7ebaea4053fd68/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:43355 "POST /session/1fd13aac053ff3a23e7ebaea4053fd68/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:43355 "POST /session/1fd13aac053ff3a23e7ebaea4053fd68/element/f.59C74E565A132A798B70C9644EC2D7EC.d.A1CE3724551D9B999496791DE9E6CC61.e.3/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:43355 "POST /session/1fd13aac053ff3a23e7ebaea4053fd68/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:43355 "POST /session/1fd13aac053ff3a23e7ebaea4053fd68/element/f.59C74E565A132A798B70C9644EC2D7EC.d.A1CE3724551D9B999496791DE9E6CC61.e.4/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:43355 "POST /session/1fd13aac053ff3a23e7ebaea4053fd68/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:43355 "POST /session/1fd13aac053ff3a23e7ebaea4053fd68/element/f.59C74E565A132A798B70C9644EC2D7EC.d.A1CE3724551D9B999496791DE9E6CC61.e.7/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:43355 "GET /session/1fd13aac053ff3a23e7ebaea4053fd68/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:43355 "GET /session/1fd13aac053ff3a23e7ebaea4053fd68/cookie HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /crumbIssuer/api/json HTTP/1.1" 200 156
DEBUG    core.jenkins_utils:jenkins_utils.py:31 update_crumb: 1ef40a4c49c1f09c1e0563e684335b47c7b667eb1fb13dfbe0a8a49b0f2b4b10
DEBUG    LoginPage:login_page.py:32 login crumb: 1ef40a4c49c1f09c1e0563e684335b47c7b667eb1fb13dfbe0a8a49b0f2b4b10
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:43355 "POST /session/1fd13aac053ff3a23e7ebaea4053fd68/elements HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:43355 "GET /session/1fd13aac053ff3a23e7ebaea4053fd68/element/f.59C74E565A132A798B70C9644EC2D7EC.d.6D07A41CA00B0EED86DB4C09859C675A.e.70/text HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:43355 "GET /session/1fd13aac053ff3a23e7ebaea4053fd68/element/f.59C74E565A132A798B70C9644EC2D7EC.d.6D07A41CA00B0EED86DB4C09859C675A.e.71/text HTTP/1.1" 200 0
DEBUG    MainPage:ui_element.py:126 Navigating to FolderPage with click on (('link text', 'Test_Folder'), 'Test_Folder')
DEBUG    MainPage:ui_element.py:28 waiting for element_to_be_clickable(('link text', 'Test_Folder'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:43355 "POST /session/1fd13aac053ff3a23e7ebaea4053fd68/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:43355 "POST /session/1fd13aac053ff3a23e7ebaea4053fd68/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:43355 "GET /session/1fd13aac053ff3a23e7ebaea4053fd68/element/f.59C74E565A132A798B70C9644EC2D7EC.d.6D07A41CA00B0EED86DB4C09859C675A.e.71/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:43355 "POST /session/1fd13aac053ff3a23e7ebaea4053fd68/element/f.59C74E565A132A798B70C9644EC2D7EC.d.6D07A41CA00B0EED86DB4C09859C675A.e.71/click HTTP/1.1" 200 0
DEBUG    FolderPage:base_page.py:23 Waiting for page content to load: http://localhost:8080/job/Test_Folder/
DEBUG    FolderPage:ui_element.py:28 waiting for presence_of_element_located(('css selector', 'a[href*="newJob"]'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:43355 "POST /session/1fd13aac053ff3a23e7ebaea4053fd68/element HTTP/1.1" 200 0
DEBUG    FolderPage:base_page.py:26 Page content loaded: http://localhost:8080/job/Test_Folder/
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:43355 "POST /session/1fd13aac053ff3a23e7ebaea4053fd68/elements HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:43355 "GET /session/1fd13aac053ff3a23e7ebaea4053fd68/element/f.59C74E565A132A798B70C9644EC2D7EC.d.04329F8D0CEA3FDC8049A838BAD94A6E.e.93/text HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:43355 "DELETE /session/1fd13aac053ff3a23e7ebaea4053fd68 HTTP/1.1" 200 0